mod/metrics/lib/card/all_answer_query/fixed_company.rb

```diff
@@ -0,0 +1,36 @@
+class Card
+  class AllAnswerQuery
+    # Query researched and not-researched answers for a given company
+    class FixedCompany < AllAnswerQuery
```

Comment:
most of the AnswerQuery changes are code reorganization.  eg this instead of `FixedCompanyAllAnswerQuery`.